### PR TITLE
Regression models

### PR DESCRIPTION
**Pull request by vasiuddin7733**

### CODE REVIEW

File: tanzania/models/regression/lasso_regression_model.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+from sklearn.linear_model import Lasso
+from sklearn.model_selection import GridSearchCV
+from sklearn.metrics import mean_squared_error, r2_score
+
+def lasso_regression(X_train, X_test, y_train, y_test):
+    # Define a range of alpha values to search over
+    param_grid = {'alpha': [0.001, 0.01, 0.1, 1.0, 10, 100, 1000]}
+    
+    # Set up GridSearchCV with Lasso regression
+    grid_search = GridSearchCV(Lasso(), param_grid, cv=5, scoring='neg_mean_squared_error')
```
Comment (n0imaginati0n) [inline tanzania/models/regression/lasso_regression_model.py:10]: can add n_jobs=-1

**Review by n0imaginati0n — APPROVED** [review]
there are very strange graphs on resifual analysis. traditional way is to make X axis as predicted value (or real - not much difference) and Y axis as am error. the last graph in the yupiter notebook shows predicted vs actuial - not really clear, what to seek there and is not looking correct.

I see in model files, that they are using MSE as error criterion. We've used RMSE - a square   root from MSE - it has the same value as a target value. and smaller.

Residual Plot (for any regression model) - it is for kind or linear regression. other real model will have it different IMHO.

but as anyway this code just for store, not for present. I'm fine

File: tanzania/models/regression/lasso_regression_model.py (hunk 1 above)
Comment (vasiuddin7733) [reply tanzania/models/regression/lasso_regression_model.py:10]: sure, i add n_jobs=-1
